database/dtb.ksy

```diff
@@ -0,0 +1,127 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
+  application:
+    - Linux
+    - Das U-Boot
+  xref:
+    wikidata: Q16960371
+  license: CC0-1.0
+  encoding: ASCII
+  endian: be
+doc: |
+  Also referred to as Devicetree Blob (DTB). It is a flat
+  binary encoding of data (primarily devicetree data, although
+  other data is possible as well).
+
+  On Linux systems that support this the blobs can be accessed in
+  /sys/firmware/fdt :
+
+  - https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-firmware-ofw
+
+  The encoding of strings used in the strings block and struct block is
+  actually a subset of ASCII:
+
+  https://github.com/devicetree-org/devicetree-specification/blob/master/source/chapter2-devicetree-basics.rst
```

Comment:
1. It should point to a specific revision,
2. it can link to a concrete section by including the hashtag:

```suggestion
  <https://github.com/devicetree-org/devicetree-specification/blob/deb8f23/source/chapter2-devicetree-basics.rst#devicetree-structure-and-conventions>
```